File: AMS_基础信息组件/code/client/source/YssInformation.Support/Bi/AccountTree/Pojo/AccountTreeB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;
using FAST.Common.Service.Pojo.Base;
using FAST.Common.Service.Datastructure;

namespace YssInformation.Support.Bi.AccountTree.Pojo
{
    /// <summary>
    /// 组合关联信息pojo类
    /// </summary>
    [NodeDesc(ParentNode = "C_NODE_CODE_P", TreeNode = "C_NODE_CODE")]
    public class AccountTreeB : AuditableParamPojo
    {
        /// <summary>
        /// 节点代码
        /// </summary>
        private string c_NODE_CODE = "";

        /// <summary>
        /// 关联账户id
        /// </summary>
        private string c_IDEN_RELA = "";

        /// <summary>
        /// 父级节点
        /// </summary>
        private string c_NODE_CODE_P = "";

        /// <summary>
        /// 关联账户信息账户名称（不保存入数据库）
        /// </summary>
        private string c_OPEN_ACC_NAME = "";

        /// <summary>
        /// 关联账户信息账户账号（不保存入数据库）
        /// </summary>
        private string c_OPEN_ACC_NO = "";

        /// <summary>
        /// 关联账户信息账户开户地址（不保存入数据库）
        /// </summary>
        private string c_OPEN_ADDR = "";

        /// <summary>
        /// 关联账户信息账户类型（不保存入数据库）
        /// </summary>
        private string c_ACCOUNT_TYPE = "";

        /// <summary>
        /// 关联账户信息账户币种（不保存入数据库）
        /// </summary>
        private string c_DC_CODE = "";

        /// <summary>
        /// 关联账户信息账户开户机构（不保存入数据库）
        /// </summary>
        private string c_ORG_CODE = "";

        /// <summary>
        /// 属性: 节点代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_NODE_CODE")]
        public string C_NODE_CODE
        {
            set { c_NODE_CODE = value; }

            get { return c_NODE_CODE; }
        }

        /// <summary>
        /// 属性: 关联账户id 
        /// </summary>
        [JsonProperty(PropertyName = "c_IDEN_RELA")]
        public string C_IDEN_RELA
        {
            set { c_IDEN_RELA = value; }

            get { return c_IDEN_RELA; }
        }

        /// <summary>
        /// 属性:父级节点
        /// </summary>
        [JsonProperty(PropertyName = "c_NODE_CODE_P")]
        public string C_NODE_CODE_P
        {
            set { c_NODE_CODE_P = value; }

            get { return c_NODE_CODE_P; }
        }

        /// <summary>
        /// 属性: 关联账户信息账户名称（不保存入数据库）
        /// </summary>
        [JsonProperty(PropertyName = "c_OPEN_ACC_NAME")]
        public string C_OPEN_ACC_NAME
        {
            set { c_OPEN_ACC_NAME = value; }

            get { return c_OPEN_ACC_NAME; }
        }

        /// <summary>
        /// 属性: 关联账户信息账户账号（不保存入数据库）
        /// </summary>
        [JsonProperty(PropertyName = "c_OPEN_ACC_NO")]
        public string C_OPEN_ACC_NO
        {
            set { c_OPEN_ACC_NO = value; }

            get { return c_OPEN_ACC_NO; }
        }

        /// <summary>
        /// 属性: 关联账户信息账户开户地址（不保存入数据库）
        /// </summary>
        [JsonProperty(PropertyName = "c_OPEN_ADDR")]
        public string C_OPEN_ADDR
        {
            set { c_OPEN_ADDR = value; }

            get { return c_OPEN_ADDR; }
        }

        /// <summary>
        /// 属性: 关联账户信息账户类型（不保存入数据库）
        /// </summary>
        [JsonProperty(PropertyName = "c_ACCOUNT_TYPE")]
        public string C_ACCOUNT_TYPE
        {
            set { c_ACCOUNT_TYPE = value; }

            get { return c_ACCOUNT_TYPE; }
        }

        /// <summary>
        /// 属性: 关联账户信息账户币种（不保存入数据库）
        /// </summary>
        [JsonProperty(PropertyName = "c_DC_CODE")]
        public string C_DC_CODE
        {
            set { c_DC_CODE = value; }

            get { return c_DC_CODE; }
        }

        /// <summary>
        /// 属性: 关联账户信息账户开户机构（不保存入数据库）
        /// </summary>
        [JsonProperty(PropertyName = "c_ORG_CODE")]
        public string C_ORG_CODE
        {
            set { c_ORG_CODE = value; }

            get { return c_ORG_CODE; }
        }

       
    }
}

File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/bi/accountTreeB/pojo/AccountTreeB.java
package com.yss.ams.base.information.support.bi.accountTreeB.pojo;

import com.yss.framework.api.common.co.AuditableParamPojo;

public class AccountTreeB  extends AuditableParamPojo {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	/**
	 * 节点代码
	 */
	private String c_NODE_CODE = "";
	
	/**
	 * 父级节点
	 */
	private String c_NODE_CODE_P = "";

	/**
	 * 关联账户id
	 */
	private String c_IDEN_RELA = "";
	
	/**
	 * 关联账户信息账户名称（不保存入数据库）
	 */
	private String c_OPEN_ACC_NAME = "";
	
	/**
	 * 关联账户信息账户账号（不保存入数据库）
	 */
	private String c_OPEN_ACC_NO = "";
	
	/**
	 * 关联账户信息账户开户地址（不保存入数据库）
	 */
	private String c_OPEN_ADDR = "";
	
	/**
	 * 关联账户信息账户类型（不保存入数据库）
	 */
	private String c_ACCOUNT_TYPE = "";
	
	/**
	 * 关联账户信息账户币种（不保存入数据库）
	 */
	private String c_DC_CODE = "";
	
	/**
	 * 关联账户信息账户开户机构（不保存入数据库）
	 */
	private String c_ORG_CODE = "";
	
	public String getC_NODE_CODE() {
		return c_NODE_CODE;
	}
	public void setC_NODE_CODE(String c_NODE_CODE) {
		this.c_NODE_CODE = c_NODE_CODE;
	}
	public String getC_NODE_CODE_P() {
		return c_NODE_CODE_P;
	}
	public void setC_NODE_CODE_P(String c_NODE_CODE_P) {
		this.c_NODE_CODE_P = c_NODE_CODE_P;
	}
	public String getC_IDEN_RELA() {
		return c_IDEN_RELA;
	}
	public void setC_IDEN_RELA(String c_IDEN_RELA) {
		this.c_IDEN_RELA = c_IDEN_RELA;
	}
	public String getC_OPEN_ACC_NAME() {
		return c_OPEN_ACC_NAME;
	}
	public void setC_OPEN_ACC_NAME(String c_OPEN_ACC_NAME) {
		this.c_OPEN_ACC_NAME = c_OPEN_ACC_NAME;
	}
	public String getC_OPEN_ACC_NO() {
		return c_OPEN_ACC_NO;
	}
	public void setC_OPEN_ACC_NO(String c_OPEN_ACC_NO) {
		this.c_OPEN_ACC_NO = c_OPEN_ACC_NO;
	}
	public String getC_OPEN_ADDR() {
		return c_OPEN_ADDR;
	}
	public void setC_OPEN_ADDR(String c_OPEN_ADDR) {
		this.c_OPEN_ADDR = c_OPEN_ADDR;
	}
	public String getC_ACCOUNT_TYPE() {
		return c_ACCOUNT_TYPE;
	}
	public void setC_ACCOUNT_TYPE(String c_ACCOUNT_TYPE) {
		this.c_ACCOUNT_TYPE = c_ACCOUNT_TYPE;
	}
	public String getC_DC_CODE() {
		return c_DC_CODE;
	}
	public void setC_DC_CODE(String c_DC_CODE) {
		this.c_DC_CODE = c_DC_CODE;
	}
	public String getC_ORG_CODE() {
		return c_ORG_CODE;
	}
	public void setC_ORG_CODE(String c_ORG_CODE) {
		this.c_ORG_CODE = c_ORG_CODE;
	}
	public static long getSerialversionuid() {
		return serialVersionUID;
	}
}
